styles.css

```diff
@@ -0,0 +1,85 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  color: rgb(46, 95, 0);
+}
+
+body {
+  /* background-color: aqua; */
+  background: repeating-linear-gradient(
+    45deg,
+    rgb(203, 255, 154),
+    rgb(203, 255, 154) 20px,
+    rgb(255, 255, 240) 50px,
+    rgb(255, 255, 240) 100px
+  );
+}
+img {
+  padding-bottom: 10px;
```

Comment:
I notice that you are using px a lot in your css file. Have a look at vw and vh. These will change according to the viewport instead so they are a lot more reactive. They can be used with most properties e.g. font-size, margin. padding etc.